IBM/carbon-addons-iot-react
# [datetimepicker] bug when going from absolute to preset and clicking apply

Closes #1114 

**Summary**

The bug is when you click on custom ranges and then go back to presets. If you click apply an error is thrown from referencing an object that is undefined.  Additionally I removed the `_unstable` prefix that we discussed for our experimental components in #1137 . 

![abp](https://user-images.githubusercontent.com/22093326/81043594-24cd5680-8ed0-11ea-9de6-9d17e03f6c75.gif)


**Change List (commits, features, bugs, etc)**

- removed `_unstable` from DateTimePicker.jsx,  DateTimePicker.test.jsx, DateTimePicker.story.jsx, and index.js
- removed the resetPreset function call when you switch between preset and custom in DateTimePicker.jsx
- updated snapshots. 

**Acceptance Test (how to verify the PR)**

- Go to the DateTimePicker story
- click on custom range
- go back to preset and see one selected and if you hit apply see the app does not throw an error. 

I checked various combinations of values and applying and could not see why the function I removed was needed. Can  double check multiple values to be sure those values are being added to the input.
